src/arrayMethodIncludes.js

```diff
@@ -4,8 +4,25 @@
  * Implement method includes
  */
 function applyCustomIncludes() {
-  [].__proto__.includes2 = function(valueToFind, fromIndex) {
-    // write code here
+  [].__proto__.includes2 = function(valueToFind, fromIndex = 0) {
+    let startIndex = fromIndex;
+    const countedIndex = this.length + startIndex;
+
+    if (startIndex >= this.length) {
+      return false;
+    } else if ((startIndex < 0) && (countedIndex > 0)) {
+      startIndex = countedIndex;
+    } else if (countedIndex < 0) {
+      startIndex = 0;
+    }
```

Comment:
два наступні if нічого не мають спільного з першим блоком
```suggestion
    if (startIndex >= this.length) {
      return false;
    } 
    
    if ((startIndex < 0) && (countedIndex > 0)) {
      startIndex = countedIndex;
    } else if (countedIndex < 0) {
      startIndex = 0;
    }
```